integration/kubernetes/confidential/agent_image.bats

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bats
+# Copyright (c) 2022 Red Hat
```

Comment:
I wonder if this should have an IBM copyright as I think it's been modified from the crictl one quite a bit?

Reply:
I think so @stevenhorsman

Reply:
Awesome I've changed this now thanks both 😄